subprojects/tooling-api/src/main/java/org/gradle/tooling/LongRunningOperation.java

```diff
@@ -130,7 +130,7 @@ public interface LongRunningOperation {
      * System properties can be also defined in the build scripts (and in the gradle.properties file), or with a JVM argument. In case of an overlapping system property definition the precedence is as follows:
      * <ul>
      *     <li>{@code withSystemProperties()} (highest)</li>
```

Comment:
```suggestion
     *     <li>{@code withSystemProperties(...)} (highest)</li>
```

Reply:
Even if there is only one of these, I would align this with the `...` notation (see below).